examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,504 @@
+{
```

Comment:
<p>And now you are mixing <code>V('predictions')</code> and <code>B.predictions</code>. And the problem is not only that you get your variables from different places (pipeline/batch), but also that you reworked only a line of pipeline to be in one style, but what I meant earlier asking you to be more consistent is sticking to the same style of addressing named expression throughout the whole pipeline definition (where it's possible), and even broader — through the notebook itself. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>